easybuild/easyblocks/generic/configuremake.py

```diff
@@ -50,7 +50,7 @@ def extra_options(extra_vars=None):
         extra_vars = EasyBlock.extra_options(extra=extra_vars)
         extra_vars.update({
             'configure_cmd_prefix': ['', "Prefix to be glued before ./configure", CUSTOM],
-            'prefix_opt': ['--prefix=', "Prefix command line option for configure script", CUSTOM],
+            'prefix_opt': [None, "Prefix command line option for configure script ('--prefix=' if None)", CUSTOM],
```

Comment:
because then I need to use the default value in two places, here, and below

the whole point of this change is that there are situations (with derived easyblocks and multiple inheritance) where `prefix_opt` may not be available in `self.cfg`